src/core/psxemulator.h

```diff
@@ -179,13 +179,16 @@ class Emulator {
     typedef Setting<int, TYPESTRING("MSAA"), 1> SettingMSAA;
     typedef Setting<bool, TYPESTRING("LinearFiltering"), true> SettingLinearFiltering;
     typedef Setting<bool, TYPESTRING("KioskMode"), false> SettingKioskMode;
+    typedef Setting<bool, TYPESTRING("Mcd1Pocketstation"), true> SettingMcd1Pocketstation;
```

Comment:
Do we want these at true by default? What are the pros and cons?

Reply:
Woops, no. Those should've been defaulted to no, thought I changed these back already.